extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -309,6 +309,7 @@ export class MigrationCutoverDialog {
 		this._disposables.push(this._cancelButton.onDidClick((e) => {
 			vscode.window.showInformationMessage(loc.CANCEL_MIGRATION_CONFIRMATION, { modal: true }, loc.YES, loc.NO).then(async (v) => {
 				if (v === loc.YES) {
+					this._cancelButton.enabled = false;
```

Comment:
will the cancel button be re-enabled if the cancel operation fails?

Reply:
I suggest controlling the cancel button state further bellow (line 678) where it is set based on the migration status.

Reply:
this._cancelButton.enabled =
                                migrationStatusTextValue === MigrationStatus.Cancelling ||
				migrationStatusTextValue === MigrationStatus.Creating ||
				migrationStatusTextValue === MigrationStatus.InProgress;

removing migrationStatusTextValue === MigrationStatus.Cancelling must accomplish the task. Is removing this  affect any other condition ??